interview-questions.md

```diff
@@ -8,44 +8,47 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a gem?
 
-Your answer:
+Your answer: Gem is a library that is used in ruby to keep files and packages organized, similar to yarn and npm. You can add gem for testing in ruby
 
-Researched answer:
+Researched answer: A component that contains code, testing, documentation and gamespec. 
+RubyGems adds run-time loader to find and install gems find standardized librarys. 
 
 2. What does the acronym CRUD stand for?
 
-Your answer:
+Your answer: CRUD stands for : Create, Read, Update, Delete.
```

Comment:
Make sure to extend your answer.  Simply answering what the acronym stands for won't show an interviewer your full knowledge.  Make it conversational.